### ISSUE TITLE
Possible to expose supporting hospital data?

### ISSUE THREAD

**Issue by MoeFwacky** [DonorDrive/PublicAPI#22]
Something I should have submitted earlier. In participants, it would be very useful to be able to grab the supporting hospital from the API to more easily grab the data with our script (we have a variety of people supporting different hospitals on our stream).

**Comment by timmixell** [comment]
I can tell you right now this is not going to get addressed before next weekend. I can also tell you it's something that is absolutely on our radar.

Keep the requests coming!

**Comment by WireJunky42** [comment]
@timmixell, I am working on a personal project and would love the ability to pull in the hospital logo.  I wasn't sure if it warranted a separate issue as getting the logo would fit within the purview of this issue.

**Comment by timmixell** [comment]
Nope. When we tackle this, we'll make sure all relevant hospital info is available 👌

**Comment by MoeFwacky** [comment]
I've started looking at making updates to my script for this year. Is this on the roadmap to be in before game day this year?

**Comment by timmixell** [comment]
Hi @MoeFwacky , this is in our backlog. I don't think we'll be able to get to it this year, but it's still on our radar both from an internal use-case perspective, as well as to facilitate integrations such as yours.

**Comment by Andrew-Best** [comment]
Is there any movement on this? I would like to implement a solution for my local hospital because their contacts at Extra Life are slower than molasses at giving them any information when I would be able to harvest it significantly faster if I were able to collect all of the donations gathered for a hospital.

**Comment by Andrew-Best** [comment]
Thanks for getting back to me @timmixell , yes that's pretty much all I would need. I can use timestamps to gather up the rest of the data I need.

One of the things my local hospital was looking for was to determine at what point during certain events they hold for Extra Life do they get the most donations, with the addition of `hospitalName` and `hospitalID` at the participant level I can pretty simply compile the data together based on timestamps since I'll now know that the participant is supporting X hospital.

Would it be possible for a filter by `hospitalID` or `hospitalName` as well? So for example, a filter I can use to gather all donations or participants that are for the specified hospital?

**Comment by terrul** [comment]
I was curious if there was any updates to this?

**Comment by timmixell** [comment]
@terrul it's still in our backlog. feel free to add any additional values/filters you're looking for. thank you!

**Comment by terrul** [comment]
@timmixell I'm apart of the [Akron Extra Life Guild](https://www.extra-life.org/index.cfm?fuseaction=donorDrive.team&teamID=63294) and we were trying to get the data to share. Right now we a manually adding it up. The data we are trying to capture was just the hospital and participant.

https://www.extra-life.org/index.cfm?fuseaction=donordrive.participantList&eventID=553&hospital=5007

Not sure if there is an easy way to do it.
